KnightJ-valheim-tame-animals

jknight6 opened the pull request:
This PR is for my second procedure, How to Tame Animals in Valheim. It contains two commits to cover the procedure draft and prepare it for review.

jckinsey01 commented:
### Hello Caimile!

As far as what you specifically asked for advice on:
- Yes, I think the gerund in the title and the use of subtitles are the right move because I believe it lets the user know they are going to be learning both. I know sometimes I follow instructions that are set up similarly (but don't have the subsections) and I end up wondering if I did, in fact, do both things the title talked about.
- I think sticking with "reserving an affix" rather than "create an affix" if that's what the game uses most often. I would maybe specify as an alert that the may see the game refer to it as "creating" but that it means the same thing.
- I like the use of location first rather than the command verbs, because then I feel like I don't have to go back and check, I just automatically know where I'm meant to be doing what I'm doing. But I'm also coming into the program without an English B.S. so I don't know if this is some weird faux pas.
- I think the use of a quote for alerts is good, it makes sure they're distinguished from the rest of the text. I would maybe also italicize them to make them stand out more? Or bold them, but you also use bold text for the selections/actions the user is supposed to make/do, so it depends on how you feel about that consistency of style. 

## User scenario
- I like that your user scenario is based off of someone who is 34, I think that might not be an audience many people would pick (I could be wrong) and I think it's a sets a good baseline of digital literacy, if that makes sense. I think people in that demographic are generally pretty good at navigating computers, the internet, and games. It would be a way different procedure to make if you were to write that she was 60 or something.
- Maybe explain *why* an affix is the best way to complete her project. As someone who has never heard of howrse (but will probably recommend it to my horse-loving, vet student roommate) it's not immediately apparent to me *why* this helps her with her breeding program.

## Procedure
1. In your first step, I would maybe consider changing it to something like "navigate to Howrse.com and sign in to your profile" then have your current step one be step two.
2. In your prerequisites section, maybe give an example of how many affixes someone can have. Like "For example, if you have 60 days seniority, you can have two affixes" to better explain how it works. It could even be styled like an alert maybe? 
3. I would perhaps add a screen shot for steps 3 and 4, one for each or one for both, unless you think that navigation is very obvious because maybe your audience already knows what that page looks like.
4. In "How to apply an affix" in the alert/note, I would maybe just say "A horse can receive only one affix".

I think that's all I have for now, lmk if there's anything I can review before submission Tuesday!

jckinsey01 commented:
Hello Caimile!
As far as what you specifically asked for advice on:

Yes, I think the gerund in the title and the use of subtitles are the right move because I believe it lets the user know they are going to be learning both. I know sometimes I follow instructions that are set up similarly (but don't have the subsections) and I end up wondering if I did, in fact, do both things the title talked about.
I think sticking with "reserving an affix" rather than "create an affix" if that's what the game uses most often. I would maybe specify as an alert that the may see the game refer to it as "creating" but that it means the same thing.
I like the use of location first rather than the command verbs, because then I feel like I don't have to go back and check, I just automatically know where I'm meant to be doing what I'm doing. But I'm also coming into the program without an English B.S. so I don't know if this is some weird faux pas.
I think the use of a quote for alerts is good, it makes sure they're distinguished from the rest of the text. I would maybe also italicize them to make them stand out more? Or bold them, but you also use bold text for the selections/actions the user is supposed to make/do, so it depends on how you feel about that consistency of style.
User scenario
I like that your user scenario is based off of someone who is 34, I think that might not be an audience many people would pick (I could be wrong) and I think it's a sets a good baseline of digital literacy, if that makes sense. I think people in that demographic are generally pretty good at navigating computers, the internet, and games. It would be a way different procedure to make if you were to write that she was 60 or something.
Maybe explain why an affix is the best way to complete her project. As someone who has never heard of howrse (but will probably recommend it to my horse-loving, vet student roommate) it's not immediately apparent to me why this helps her with her breeding program.
Procedure
In your first step, I would maybe consider changing it to something like "navigate to Howrse.com and sign in to your profile" then have your current step one be step two.
In your prerequisites section, maybe give an example of how many affixes someone can have. Like "For example, if you have 60 days seniority, you can have two affixes" to better explain how it works. It could even be styled like an alert maybe?
I would perhaps add a screen shot for steps 3 and 4, one for each or one for both, unless you think that navigation is very obvious because maybe your audience already knows what that page looks like.
In "How to apply an affix" in the alert/note, I would maybe just say "A horse can receive only one affix".
I think that's all I have for now, lmk if there's anything I can review before submission Tuesday!

jckinsey01 commented:
Hello Caimile!
As far as what you specifically asked for advice on:

Yes, I think the gerund in the title and the use of subtitles are the right move because I believe it lets the user know they are going to be learning both. I know sometimes I follow instructions that are set up similarly (but don't have the subsections) and I end up wondering if I did, in fact, do both things the title talked about.
I think sticking with "reserving an affix" rather than "create an affix" if that's what the game uses most often. I would maybe specify as an alert that the may see the game refer to it as "creating" but that it means the same thing.
I like the use of location first rather than the command verbs, because then I feel like I don't have to go back and check, I just automatically know where I'm meant to be doing what I'm doing. But I'm also coming into the program without an English B.S. so I don't know if this is some weird faux pas.
I think the use of a quote for alerts is good, it makes sure they're distinguished from the rest of the text. I would maybe also italicize them to make them stand out more? Or bold them, but you also use bold text for the selections/actions the user is supposed to make/do, so it depends on how you feel about that consistency of style.
User scenario
I like that your user scenario is based off of someone who is 34, I think that might not be an audience many people would pick (I could be wrong) and I think it's a sets a good baseline of digital literacy, if that makes sense. I think people in that demographic are generally pretty good at navigating computers, the internet, and games. It would be a way different procedure to make if you were to write that she was 60 or something.
Maybe explain why an affix is the best way to complete her project. As someone who has never heard of howrse (but will probably recommend it to my horse-loving, vet student roommate) it's not immediately apparent to me why this helps her with her breeding program.
Procedure
In your first step, I would maybe consider changing it to something like "navigate to Howrse.com and sign in to your profile" then have your current step one be step two.
In your prerequisites section, maybe give an example of how many affixes someone can have. Like "For example, if you have 60 days seniority, you can have two affixes" to better explain how it works. It could even be styled like an alert maybe?
I would perhaps add a screen shot for steps 3 and 4, one for each or one for both, unless you think that navigation is very obvious because maybe your audience already knows what that page looks like.
In "How to apply an affix" in the alert/note, I would maybe just say "A horse can receive only one affix".
I think that's all I have for now, lmk if there's anything I can review before submission Tuesday!

kb-boop commented:
## Staging

Super helpful staging. I really like how you jump right into defining the vocabulary, "affix," and give clear explanation of where to find it on the screen.

I assume your prerequisites are very clear, as someone who doesn't play this game.

## Coaching

Very solid, clear coaching. No suggestions

## Alerting

Is there anything that could go wrong for the player that you could add at the end? Are there any problems you anticipate?

jknight6 committed:
Add my project and begin work on procedure one

This commit adds the project folders for my procedures, and adds the first sections of my first procedure, How to Defeat Bonemass in Valheim.

jknight6 committed:
Finish writing procedure one

This commit finishes the first draft for procedure one and prepares it for review.

jknight6 committed:
Images, About and Prerequisites

This commit covers the images for this procedure, as well as the About and Prerequisites section.

jknight6 committed:
How to of the procedure

This commit wraps up the procedure with the how-to steps and conclusion.

jknight6 commented:
This PR is ready for review.

Alli-Wray commented on knightj/p-valheim-tame-animals.md:
```diff
@@ -0,0 +1,58 @@
+1. Prepare the enclosure
+    - For boar, this can be a simple wooden fence.
+    - For lox and wolves, you can use your pickaxe to dig a deep pit that the animal can't climb out of. If you have the materials, you can also build an enclosure that is stronger than a simple wooden fence out of stone or iron.
+2. Lure the animal to the enclosure in one of the following ways:
```
Are there any warnings/alerts that can be made here? Can the animals cause physical harm to the user in the game? Will the harpoon hurt/kill the animal if used incorrectly?

jknight6 replied:
Yes, I added a warning for that.

Alli-Wray commented on knightj/p-valheim-tame-animals.md:
```diff
@@ -0,0 +1,58 @@
+
+>_**During this time you need to be close to the animal, but not close enough to alert it.**_
+
+You know that it is being tamed when yellow hearts appear above the animal.
```
The success condition/desired outcome here is well done. I do think this line could be combined with line 48 to make the coaching here a bit more concise/coherent.

jknight6 replied:
Agreed

Alli-Wray commented on knightj/p-valheim-tame-animals.md:
```diff
@@ -0,0 +1,58 @@
+
+After the animal is enclosed and has food, it takes about 30 minutes for them to be tamed. 
+
+>_**During this time you need to be close to the animal, but not close enough to alert it.**_
```
Perhaps adding a more abrupt cue for this warning/alert would be helpful. I didn't really grasp that it was an important note when initially reviewing the preview for this procedure.

jknight6 replied:
I changed this and the following sentence into one alerting event, I think that helps.

Alli-Wray commented on knightj/p-valheim-tame-animals.md:
```diff
@@ -0,0 +1,58 @@
+
+To tame a lox, you need the ability to survive in the plains, either a pickaxe or strong building materials like stone or core wood, and one of the foods found in the plains.
+
+## How to Tame Animals
```
Adding an imperative verb here could be better for coaching - "Tame your Animals" rather than "How to Tame Animals" maybe?

jknight6 replied:
Agreed

Alli-Wray reviewed (commented):
I've left a few suggestions. Like my other two reviews, I found that the staging and coaching is very well done, but some more noticeable alerting might be helpful if it is applicable to certain steps.

stevesynk commented:
This one had good organization. I liked how you addressed each specific animal in the pre-reqs, but then made a general set of instructions with references to specific animals when necessary.

I had some of the same questions as Alli (see line-specific comments), but my other main issue is that the images are not appearing to me when I view the preview of the procedure. I can however see the images in the images folder. Not sure if I'm doing something wrong, or if perhaps the syntax is off?

jknight6 committed:
Edits based on peer review

This commit contains edits based on peer review comments.

lingeringcode committed:
Feedback from Lindgren
